src/main/java/pl/kzcwat/localincidentserver/region/RegionRepository.java

```diff
@@ -0,0 +1,16 @@
+package pl.kzcwat.localincidentserver.region;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface RegionRepository extends CrudRepository<RegionEntity, Long> {
```

Comment:
```suggestion
public interface RegionRepository extends JpaRepository<RegionEntity, Long> {
```
Jeśli dobrze pamietam to CrudRepository nie ma możliwości stronicowania